tools/qontract_cli.py

```diff
@@ -1760,6 +1765,64 @@ def app_interface_merge_history(ctx):
     print_output(ctx.obj["options"], merge_queue_data, columns)
 
 
+@get.command(
+    short_help="obtain a list of all resources that are managed "
+    "on a customer cluster via a Hive SelectorSyncSet."
+)
+@click.pass_context
+def selectorsyncset_managed_resources(ctx):
+    vault_settings = get_app_interface_vault_settings()
+    secret_reader = create_secret_reader(use_vault=vault_settings.vault)
+    clusters = get_clusters()
+    oc_map = init_oc_map_from_clusters(
+        clusters=clusters,
+        secret_reader=secret_reader,
+        integration="qontract-cli",
+        thread_pool_size=1,
+        init_api_resources=True,
+    )
+    columns = [
+        "cluster",
+        "SelectorSyncSet_name",
+        "SaaSFile_name",
+        "kind",
+        "namespace",
+        "name",
+    ]
+    data = []
+    for c in clusters:
+        c_name = c.name
+        if c_name != "hivei01ue1":
```

Comment:
why exclude this hive shard and not all the others as well?

Reply:
I think some selectorsyncsets apply on hive shards... inception kind of thing